.github/workflows/check_coding_standards.yml

```diff
@@ -0,0 +1,30 @@
+name: Coding Standards
+
+on:
```

Comment:
Have you tried the action also with this syntax?
(The loose `    - pull_request` have to be deleted)

```suggestion
on: pull_request
```

Reply:
I'll try. That would be crazy if that worked :D

Reply:
It didn't help, still same git error.

Also tried 2.5, 2.1 and dev-master, still same error.

Probably something with GIT changed in workflows.

Reply:
You what si also weird?

That commit with fixes is run before the fixing command even finishes!

![image](https://user-images.githubusercontent.com/924196/71924147-b3360380-318e-11ea-9315-0e68b62c02bd.png)